components/CreateConfirm/index.tsx

```diff
@@ -29,29 +29,19 @@ export const CreateConfirm = () => {
 
   // Get environment variables
   const { NEXT_PUBLIC_EXPLORER_URL: explorerUrl } = process.env
-  const { NEXT_PUBLIC_LOTUS_NODE_JSONRPC: apiAddress } = process.env
 
   // Get the message by cid
-  const {
-    data: messageData,
-    error: messageError,
-    loading: messageLoading
-  } = useMessageQuery({
+  const { data: messageData, error: messageError } = useMessageQuery({
```

Comment:
The `messageLoading` boolean was only passed as a dependency to the `useEffect` below, but not used